### PR TITLE
Args typing in register

### PR DESCRIPTION
**Pull request by mamsdeveloper**
Add typings for *args and **kwargs in DialogRegistry.register

### CODE REVIEW

**Comment by vlkorsakov** [comment]
They are `Any` by default, typehint for what?

**Comment by mamsdeveloper** [comment]
> They are `Any` by default, typehint for what?

Pylance (pyright) linter recognize it as Unknown 
image

File: src/aiogram_dialog/api/entities/stack.py [hunk 1]
```diff
@@ -43,6 +43,7 @@ class Stack:
     last_income_media_group_id: Optional[str] = field(
         compare=False, default=None,
     )
+    key: Optional[str] = field(compare=False, default=None)
```
Comment (Tishka17) [inline src/aiogram_dialog/api/entities/stack.py:46]: can't see if it is somehow used

Comment (mamsdeveloper) [reply src/aiogram_dialog/api/entities/stack.py:46]: Sorry, "add key field" is unwanted commit that I made for project purpose. Only arguments typing commit matter

**Comment by Tishka17** [comment]
Thanks, I've copied to actual refactored version

**Pull request closed without merging**